Source/NSubstitute/Exceptions/CouldNotSetReturnException.cs

```diff
@@ -45,6 +45,17 @@ public class CouldNotSetReturnDueToNoLastCallException : CouldNotSetReturnExcept
 #if NET35 || NET4 || NET45
     [Serializable]
 #endif
+    public class CouldNotSetReturnDueToMissingInfoAboutPreviousCallException : CouldNotSetReturnException
+    {
+        public CouldNotSetReturnDueToMissingInfoAboutPreviousCallException() : base("Could not find information about the previous call to return from.") { }
```

Comment:
There's already an exception that meant for that: `CouldNotSetReturnDueToNoLastCallException`
And most probably users want to see a better error message. Something like here: https://github.com/nsubstitute/NSubstitute/blob/f3608f0a55cabcc78c50f33466d867b537e19405/Source/NSubstitute/Exceptions/CouldNotSetReturnException.cs#L11

Also "the previous" call is a bit misleading for me.